openapi.yml

```diff
@@ -342,6 +342,11 @@ components:
         valueAt:
           description: URL or other pointer to the location of the contributor information.
           type: string
+        parallelContributor:
+          description: For multiple representations of infomration about the same contributor (e.g. in different languages).
```

Comment:
I think there's a typo on this line: `s/infomration/information/`

Reply:
Fixed.